sass/_content.scss

```diff
@@ -5,287 +5,319 @@
+    padding-bottom: 2px;
 
-        &:hover {
-            border-color: #fff;
-            color: #fff;
-        }
+    &:hover {
+      border-color: transparent;
+      color: $color-white;
     }
+  }
 
-    .entry-subtitle a {
-        color: #aaa;
+  .entry-title a:hover {
+    border-color: $color-white;
+  }
 
-        &:hover {
-            border-color: #999;
-            color: #aaa;
-        }
-    }
+  .entry-meta a,
+  .entry-summary a {
+    border-color: $color-white;
 
-    * {
-        color: #fff;
+    &:hover {
+      border-color: transparent;
     }
+  }
 
-    &:after {
-        display: none;
+  .entry-subtitle a {
+    color: #aaa;
+
+    &:hover {
+      border-color: #999;
+      color: #aaa;
     }
-}
+  }
 
-.hentry {
-	clear: both;
-	margin: 0;
-    padding: 50px 0;
-    position: relative;
+  a.author {
+    border-color: transparent;
 
-    &:after {
-        background: rgba(0, 0, 0, .15);
-        bottom: 0;
-        content: '';
-        height: 1px;
-        left: 0;
-        position: absolute;
-        right: 0;
-        @include site-width;
+    .name {
+      border-color: $color-white;
     }
 
-    .single &:after {
-        display: none;
+    &:hover {
+      .name {
+        border-color: transparent;
+      }
     }
+  }
 
-	.single &,
-	.page & {
-        border: 0;
-        padding: 0;
-    }
+  * {
+    color: #fff;
+  }
 
-    .cover-background {
-      display: none;
-    }
+  code {
+    background: $color-dark-grey-alt;
+  }
 
-    &.sticky .cover-background {
-      display: block;
-    }
+  &:after {
+    display: none;
+  }
+}
 
-    &:first-of-type {
-        margin-top: 77px; // header height
+.hentry {
+  clear: both;
+  margin: 0;
+  padding: 50px 0;
+  position: relative;
+
+  &:after {
+    background: rgba(0, 0, 0, .15);
+    bottom: 0;
+    content: '';
+    height: 1px;
+    left: 0;
+    position: absolute;
+    right: 0;
+    @include site-width;
+  }
 
-        .featured-container + & {
-            margin-top: 40px;
-        }
+  .single &:after {
+    display: none;
+  }
 
-        .cover ~ & {
-            margin-top: 60px;
-        }
+  .single &,
+  .page & {
+    border: 0;
+    padding: 0;
+  }
 
-        .infinite-wrap & {
-            margin-top: 0; // remove margin for subsequent infinite scroll pages
-        }
+  .cover-background {
+    display: none;
+  }
+
+  &.sticky .cover-background {
+    display: block;
+  }
+
+  &:first-of-type {
+    margin-top: 77px; // header height
+
+    .featured-container + & {
+      margin-top: 40px;
+    }
+
+    .cover ~ & {
+      margin-top: 60px;
     }
+
+    .infinite-wrap & {
+      margin-top: 0; // remove margin for subsequent infinite scroll pages
+    }
+  }
 }
 
 .entry {
-    &-header {
-        @include site-width;
-        position: relative;
+  &-header {
+    @include site-width;
+    position: relative;
 
-        .span {
-            color: #b6b6b6;
-        }
+    .span {
+      color: #b6b6b6;
```

Comment:
Color literals like `#b6b6b6` should only be used in variable declarations; they should be referred to via variable everywhere else.